Topics/Teamwork.md

```diff
@@ -12,3 +12,25 @@ An important first step towards conflict resolution is learning how to avoid the
 [Resolving Conflicts](https://ca.indeed.com/career-advice/career-development/conflict-resolution-at-work)
 
 It is not always possible to avoid conflict. In work environment when people are exchanging ideas and working towards goals, a difference of opinion is inevitable. However, if handled properly, these conflicts can be resolved while keeping the negative impacts at a minimum, and even help create a better solution for the project at hand. This article covers ways that the parties in a conflict can help each other understand their own points of view, while keeping discourse respectful. It emphasizes ways that team members can help each other work towards a common goal.
+
+
+
+
+
+## **Roles in a Software Team **
+
+1. Project Manager: The project manager is responsible for planning and overseeing the software development process. They set goals and deadlines, allocate resources, manage the budget, and communicate with stakeholders.
+2. Software Developer: The software developer writes, tests, and maintains the software code. They work with the project manager and other team members to ensure that the software meets the project requirements and is delivered on time.
```

Comment:
```suggestion
2. Software Developer: The software developer writes and maintains the software code. They work with the project manager and other team members to ensure that the software meets the project requirements and is delivered on time. While developers may test their code during the development process, it is important to note that comprehensive testing is typically the responsibility of the quality assurance (QA) team.
```
You wrote "The software developer writes, tests, and maintains the software code." While it is true that software developers do test their code during development, it is important to note that comprehensive testing is typically the responsibility of the quality assurance (QA) team.

To avoid any potential confusion, I've suggested a minor change to the text. Specifically, I've removed "tests" from the sentence and added a brief explanation that developers may test their code during development but that comprehensive testing is the responsibility of the QA team.